pvacseq/lib/main.py

```diff
@@ -28,6 +35,11 @@ def main(args_input = sys.argv[1:]):
                         help="length of subpeptides(epitopes) to predict ; Multiple lengths can be specified using a comma-separated list. Typical epitope lengths vary between 8-11.",
                         type=lambda s:[int(epl) for epl in s.split(',')]
                         )
+    parser.add_argument("prediction_algorithms",
+                        choices=['NetMHCpan', 'NetMHC', 'SMMPMBEC', 'SMM', 'NetMHCcons', 'PickPocket'],
```

Comment:
It seems a little odd that these are harcoded here, as well as in `prediction_method_lookup`. Is it possible to generate the list of choices here based on the keys in the dict?